src/FileTrait.php

```diff
@@ -136,11 +136,14 @@ public function fileCleanAll(AfterScenarioScope $scope): void {
   }
 
   /**
-   * Delete managed files defined by provided properties.
+   * Delete managed files defined by provided properties/fields.
+   *
+   * Example: filename, uri, status, uid and more see Drupal\file\Entity\File.
+   *
    *
    * @code
    * Given no managed files:
-   * | filename      |
+   * | fieldName     |
```

Comment:
please revert this change and add a second `@code/@endcode` with another example of using a different field name - `path`.